apps/dashboard/shared/components/dropdowns/HeaderDAOSidebarDropdown.tsx

```diff
@@ -52,6 +55,9 @@ export const HeaderDAOSidebarDropdown = () => {
         ),
         href: `/${DaoIdEnum.UNISWAP.toLowerCase()}`,
         name: DaoIdEnum.UNISWAP,
+        isDisabled:
+          daoConfigByDaoId[DaoIdEnum.UNISWAP].supportStage ===
+          SupportStageEnum.ANALYSIS,
```

Comment:
This doesn't seem a scalable solution imho, too much manual work every time a new DAO is added

Reply:
```suggestion
        isDisabled:
          daoConfigByDaoId[daoId].supportStage ===
          SupportStageEnum.ANALYSIS,
```